Cargo.toml

```diff
@@ -39,10 +39,11 @@ apca = "0.24.0"
 anyhow = {version = "1.0", default-features = false, features = ["std"]}
 chrono = {version = "0.4", default-features = false}
 chrono-tz = {version = "0.6.1", default-features = false, features = ["std"]}
+clap = { version = "3.1.18", features = ["derive", "env"] }
+clap_complete = { version = "3.1.4" }
```

Comment:
Given that this crate is only needed by the `shell-complete` utility program (which not everybody may want/need), I think we should make this dependency optional and then adjust the binary to [require it](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-required-features-field).

Reply:
Done.